development/tests/test_log_utils.py

```diff
@@ -1,18 +1,26 @@
-
 from django.test import TestCase
 from parameterized import parameterized
 
 from development.common.logs_utils import FilterLogs
 from development.constants import (
+    ALL_ACTIONS,
     ALL_STATES,
     INVALID_STATE,
-    VALID_STATE
+    MOVE_ACTION,
+    SHOOT_ACTION,
+    VALID_STATE,
+    WALL_ACTION,
 )
 from development.tests.log_scenarios import (
+    logs_test_for_kind_of_move_action,
+    logs_test_for_kind_of_shoot_action,
+    logs_test_for_kind_of_move_and_shoot_actions,
+    logs_test_for_kind_of_move_shoot_and_wall_actions,
+    logs_test_for_kind_of_move_without_actions,
     logs_test_for_validating_moves_0,
     logs_test_for_validating_moves_1,
     logs_test_for_validating_moves_2,
-    logs_test_for_validating_moves_3
+    logs_test_for_validating_moves_3,
```

Comment:
could these names be like the logs_..._kind above to be more descriptive?
`logs_test_for_kind_of_move_action`